blueprints/component-test/files/tests/__testType__/__path__/__test__.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * <%= capitalizedTestType %> test for the <%= dasherizedModuleName %> component
+ */
+
+import {expect} from 'chai'
+import {describeComponent, it} from 'ember-mocha'
+<% if (testType === 'integration' ) { %>import hbs from 'htmlbars-inline-precompile'
+import {$hook, initialize} from 'ember-hook'
```

Comment:
could we rename the `initialize` function here, just to make it less ambiguous.
So `initialize as initializeHook` and update the call below ofcourse
We have been doing that for other addons as well during their LTS upgrades.